packages/web-components/src/components/footer/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,178 @@
+import { Preview, Props, Description, Story } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import corsproxy from '../../../../../../docs/cors-proxy.md';
+
+# Footer
+
+> The footer component is a required navigational pattern for IBM.com that
+> displays consistently at the bottom of each page.
+
+> 💡 Check our
+> [CodeSandbox](https://githubbox.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/web-components/examples/codesandbox/components/footer)
+> example implementation.
+
+## Getting started
+
+### Quick start
+
+`<dds-footer-container>` automatically renders contents of footer from default service data. If you want change in services/data integration, etc., see ["Customizing data integration" section](#customizing-data-integration) below.
+
+##### JS
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/footer/footer-container';
+```
+
+##### HTML
+
+```html
+<dds-footer-container></dds-footer-container>
```

Comment:
Is there an option to choose the short footer?

Reply:
Yes, `<dds-footer-container>` supports all attributes/properties that `<dds-footer-composite>` supports, given `<dds-footer-container>` is `<dds-footer-composite>` with the default Redux store mixed in. Made an update to reflect that.

Reply:
Thanks, can there be a separate storybook story that shows the short footer? Also might help to show in the quick start section how to configure that because I think a lot of adopters will be reaching for that.

Reply:
Makes sense @jeffchew - Updated.